ios/NativeViews/TextileImageView.m

```diff
@@ -318,16 +318,18 @@ - (void)reloadImage
 //                                partialLoadBlock:partialLoadHandler
 //                                 completionBlock:completionHandler];
 
-    NSError *error;
-    UIImage *image;
-    NSString *base64String = [_bridge.textileNode _getHashData:source.hashPath error:&error];
-    if (base64String) {
-      NSString *finalBase64String = [@"data:image/jpeg;base64," stringByAppendingString:base64String];
-      NSURL *url = [NSURL URLWithString:finalBase64String];
-      NSData *imageData = [NSData dataWithContentsOfURL:url];
-      image = [UIImage imageWithData:imageData];
-    }
-    completionHandler(error, image);
+    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
Added this to allow the image data request to enter another thread. https://github.com/facebook/react-native/issues/14841

Reply:
Yea, cool. This makes sense. Now that we're calling these methods directly from UIKit code (our native image view), the method is being called on the main thread and blocking the UI. Previously, any method in here was just called from the JS layer, and not on the main thread.

Reply:
Oh, just realized this is actually in the native image class... Cool, this works!